docs/data/charts/radar/radar.md

```diff
@@ -93,7 +93,16 @@ Otherwise, the other area would catch the pointer event, making it impossible to
 
 To remove highlight, set the `highlight` prop to `'none'`.
 
-## Tooltip 🚧
+## Tooltip
+
+Like other chart, the radar chart [tooltip](/x/react-charts/tooltip/) can be customized with slots.
+The `slots.tooltip.trigger` allows to switch between:
+
+- `'item'`—when the user's mouse hovers over a radar area, the tooltip displays data about this series.
+- `'axis'`—the user's mouse position is associated with a metric. The tooltip displays data about all series along this specific metric.
```

Comment:
We mention the 'item' option first, but the demo shows the 'axis' option first. 

We should probably follow the same order. Additionally, it make sense to add a small title above each chart with the tooltip trigger being used to make it clearer.